src/data-grid/data-grid-cell.styles.ts

```diff
@@ -50,4 +50,9 @@ export const dataGridCellStyles = (
 		color: ${listActiveSelectionForeground};
 		outline: none;
 	}
+	:host(:${focusVisible}) ::slotted(*),
+	:host(:focus) ::slotted(*),
+	:host(:active) ::slotted(*) {
+		color: ${listActiveSelectionForeground} !important;
```

Comment:
The `!important` is needed because I discovered that (for whatever reason) a lot of base HTML tags (i.e. `a`, `code`, etc.) are styled within a `style` tag that lives directly within the webview HTML document.

As a result, without the `!important` the styles within the `style` tag will have higher specificity than styles defined in the code above and get overridden.

I think this is also the reason that I had such a ridiculously challenging time trying to fix these bugs back in November because I didn't catch that these base tags were being styled directly in the HTML document and needed an `!important` tag to be overridden.

Reply:
For the purposes of posterity, here are some reference screenshots to what I'm talking about in case we ever run into this again.

_Note how the selector for `a, a code` has higher specificity than the styles defined by the `data-grid-cell`. It's only by applying `!important` that we can override these higher priority styles._

<img width="742" alt="Screen Shot 2022-03-02 at 2 34 43 PM" src="https://user-images.githubusercontent.com/39639992/156461548-1345b024-9964-47d2-bd87-e0134aa5cd3e.png">

_By clicking the `<style>` link in the top-right corner of the styling block in the above image, the DevTools inspector will take us to the spot where these styles are applied and here we can see that it is an inline `style` tag that lives in the `head` of the webview iframe._

<img width="884" alt="Screen Shot 2022-03-02 at 2 41 05 PM" src="https://user-images.githubusercontent.com/39639992/156462038-80ba5c5e-f731-4f42-a59c-15badcc88348.png">